apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -863,68 +893,82 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
-                                        You can add new attributes and mappings by navigating to
-                                            <Link
-                                                external={ false }
-                                                onClick={ () => {
-                                                    history.push(
-                                                        AppConstants.getPaths()
-                                                            .get("LOCAL_CLAIMS")
-                                                    );
-                                                } }
-                                            > Attribute.
-                                            </Link>
-                                        </Trans>
-                                    </Hint>
-                                )
-                            }
+                                    <Link
+                                        external={ false }
+                                        onClick={ () => {
+                                            history.push(
+                                                AppConstants.getPaths()
+                                                    .get("ATTRIBUTE_MAPPINGS")
+                                                    .replace(":type", ClaimManagementConstants.OIDC)
+                                            );
+                                        } }
+                                    > 
+                                        Attributes.
+                                    </Link>
+                                </Trans>
+                            </Hint>
+                        </Grid.Column>
+                    </Grid.Row>
+                    <Grid.Row className="mt-5">
+                        <Grid.Column>
+                            <Grid.Row>
+                                <Form>
+                                    <Form.Field>
+                                        <div className="display-flex">
+                                            <CopyInputField
+                                                className="copy-input spaced"
+                                                value={ selectedScopes.join(" ") }
+                                                data-componentid={ `${ componentId }-selected-scope-area` }
+                                            />
+                                        </div>
+                                    </Form.Field>
+                                </Form>
+                            </Grid.Row>
                         </Grid.Column>
                     </Grid.Row>
+                    <Hint>
+                        <Trans
+                            i18nKey={
+                                "console:develop.features.applications.edit.sections.attributes." +
+                                                "selection.selectedScopesComponentHint"
```

Comment:
Can we have one tab?
```suggestion
                                           "selection.selectedScopesComponentHint"
```